### PR TITLE
[docs] add linux amd64 download button

### PR DESCRIPTION
**Pull request by cgardens**
## What
* Replaces links to releases page with buttons that just download the relevant binaries.

## How
* Add a client module script that figures out which architecture a button is for and grabs the right link.

## Review guide
1. `docs/using-airbyte/getting-started/oss-quickstart.md`
2. `docusaurus/src/scripts/download-abctl-buttons.js`
3. `docusaurus/docusaurus.config.js`

## User Impact
* Better UX for downloading abctl binaries.

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: docusaurus/src/scripts/download-abctl-buttons.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            }
+          }
+        });
+      }
+    }
+  }
+
+  window.addEventListener('load', initializeAllDownloadButtons);
+  const observer = new MutationObserver(handleMutations);
+  observer.observe(document.body, { childList: true, subtree: true });
```
Comment (cgardens) [inline docusaurus/src/scripts/download-abctl-buttons.js:61]: There must be a less hacky way of doing this.

Comment (cgardens) [reply docusaurus/src/scripts/download-abctl-buttons.js:63]: Was fighting with the fact that on window load seemed to trigger at a time such that the script ran too early.

Comment (colesnodgrass) [reply docusaurus/src/scripts/download-abctl-buttons.js:63]: Might want to ask the compose team for any pointers.

Comment (cgardens) [reply docusaurus/src/scripts/download-abctl-buttons.js]: eh. it's not a typical javascript / dom question, which is why i don't want to bug them. It's a nuance of docusaurus. dug a little more and got to an answer I feel okay about.

**Review by bgroff — APPROVED** [review]

**Pull request merged**